server/controllers/prepMaterialTeacherController.js

```diff
@@ -0,0 +1,146 @@
+var formidable = require("formidable");
+const fs = require("fs");
+var dir = __dirname + "/../uploads/qnaStudents/";
+const { TeacherPrepMaterial } = require("../models/prepMaterialTeacher.model");
+const { mongo } = require("mongoose");
+
+const path = require("path");
+const responseHelpers = require("../helpers/response.helpers");
+
+// API to post Practice Quiz Test from teacher side login  :
+exports.postCreateQuiz = async (req, res, next) => {
+  if (req.user.role == "teacher") {
+    return responseHelpers.errorMessage(
+      { message: "Only teachers are permitted to Create a Quiz : " },
+      res,
+      401
+    );
+  }
+
+  var form = new formidable.IncomingForm();
+  form.parse(req, (err, fields) => {
+    if (err) {
+      responseHelpers.errorMessage(err, res, 400);
+    }
+
+    var insertCreateQuizData = {
```

Comment:
why `var` , why not `let` or `const`?

Reply:
sure .. let me make the fixes and I will push by EOD today .

Reply:
Updated the PR.